docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -638,6 +638,70 @@ Avaya
 Avaya bought Nortel's wired networks assets. So Avaya switches are, in effect, re-branded 
 Nortels. See <<_nortel,Nortel section>> of this document for configuration instructions. 
 
+802.1X with MAC Authentication bypass and VoIP
+++++++++++++++++++++++++++++++++++++++++++++++
+
+Global config settings:
+
+  sntp server primary address 192.168.1.5
+  sntp enable
+  radius server host 192.168.1.5 acct-enable 
+  radius server host key useStrongerSecret
+  radius server host key useStrongerSecret used-by eapol
+  radius server host key useStrongerSecret used-by non-eapol
+  radius dynamic-server client 192.168.1.5
+  radius dynamic-server client 192.168.1.5 secret useStrongerSecret
+  radius dynamic-server client 192.168.1.5 enable
+  radius dynamic-server client 192.168.1.5 process-change-of-auth-requests
+  radius dynamic-server client 192.168.1.5 process-disconnect-requests
+
+  vlan create 2,3,4,5 type port
```

Comment:
We use the assumptions in the beginning of this guide. So you have to RTFM first.